index.js

```diff
@@ -26,44 +26,44 @@ function sniff(buffer, callback) {
         if (header.indexOf('\"type\":') !== -1) {
             var m = /"type":\s?"(.+?)"/.exec(header);
             if (!m) {
-              return callback(invalid('Unknown filetype'));
+                return callback(invalid('Unknown filetype'));
```

Comment:
Because no `tabwidth` is defined and the default is `4` 
https://github.com/mapbox/mapbox-file-sniff/blob/master/.eslintrc
http://eslint.org/docs/rules/max-len#tabwidth